client/js/actions/snippet-actions.js

```diff
@@ -15,6 +15,12 @@ class SnippetActions {
       .catch(err => this.dispatch({ok: false, error: err}));
   }
 
+  getAllOfCurrentUser() {
+    axios.get('/snippets/user')
+      .then(res => this.dispatch({ok: true, snippets: res.data}))
+      .catch(err => this.dispatch({ok: false, error: err}));
```

Comment:
maybe use `FlashMessages.pushMessage` for displaying errors? 
example: https://github.com/binarapps/snippet-hero/blob/master/client/js/actions/user-actions.js#L12-L15